### PR TITLE
Travis trigger script in python added - (Solves- #124)

### LINKED ISSUE #124: Trigger Travis using Python instead of script file
**I'm submitting a ...**  (check one with "x")
- [X] bug report

**Actual Behaviour**
Currently, the script.sh file is used to trigger travis.

This creates these problems:
- Python itself is a scripting language. We use two languages here
- We set environment variables in the app.py to call the script. This could be python arguments to a function (cleaner code)
- As I understand, the redirect to /output has the user trigger the build. When two users use this app at the same time, two identical builds may be started. (race condition)

**Expected Behaviour**

- use python to start the travis build on POST

Proposed steps:
  - add a file build.py which can be imported because this behavior should not be in the app.py so one can import and test it independently.
  - add a function to trigger the build

**Steps to reproduce it**

**Other information:**

- you can use the requests module or the urllib module to trigger for http requests.

**Would you like to work on the issue?**

Not now but later, yes.

### PR DESCRIPTION
**Pull request by pranav1698**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [ ] The unit tests pass locally with my changes (provide a screenshot or link for test) 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
This adds alternative python script for the triggering of travis script 

#### Changes proposed in this pull request:

- Addition of alternative script build.py for the triggering of travis build which contains a method to send the trigger request
- The method is called from app.py with the required arguments
-

Fixes #124

### CODE REVIEW

**Comment by pranav1698** [comment]
Hosted site: https://boiling-thicket-10511.herokuapp.com/

**Comment by tabesin** [comment]
(I mean, after all @niccokunzmann proposed this change.)

**Comment by pranav1698** [comment]
Can you add a screenshot

**Comment by rpotter12** [comment]
@pranav1698 now it's working.

File: app.py [hunk 1]
```diff
@@ -118,7 +119,7 @@ def index():
 def output():
     if flag:
         if os.environ['TRAVIS_TAG']:  # if TRAVIS_TAG have value it will proceed
-            os.system('./script.sh')
```
Comment (tabesin) [inline app.py]: In principle you could also comment that out. 

alright....!

File: app.py [hunk 2]
```diff
@@ -118,7 +119,7 @@ def index():
...
+            build.send_trigger_request(os.environ['email'], os.environ['TRAVIS_TAG'], os.environ['event_url'],os.environ['TRAVIS_SCRIPT'])
             print ('/build called')
```
Comment (xeon-zolt) [inline app.py]: let's remove this print too this was there for debugging the Heroku logs very old and not useful fine

Comment (pranav1698) [reply app.py]: @xeon-zolt could you also please describe the use of travis_script_1, we can use the travis_tag to the server in other ways

**Pull request merged**